lib/couchdb/index.js

```diff
@@ -84,23 +85,56 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findOneEqById = R.curry((couch, bucket, predicates) =>
```

Comment:
I think you should have it `_findOneByIdWhereEq :: Couch -> Bucket -> Id -> Predicates -> Record`

This is so it's explicit that you need an ID and will then filter by the predicates